app/controllers/Management.scala

```diff
@@ -40,10 +43,23 @@ object Management extends Controller with LazyLogging {
     override def ok = exec()
   }
 
+  class EitherTest[A](name: String, exec: () => List[String] \/ A) extends Test {
+    override def messages = exec().swap.toOption.toList.flatten
+    override def ok: Boolean = exec().isRight
```

Comment:
do you really want it to do the exec twice if it fails? might need to put the result of exec into a lazy val to avoid reevaluating it